StepLang.Tooling.Formatting/StepLang.Tooling.Formatting.csproj

```diff
@@ -7,4 +7,14 @@
         <RootNamespace>StepLang.Formatters</RootNamespace>
     </PropertyGroup>
 
+
```

Comment:
This space intentionally left blank

Reply:
I'm not sure why this is. I noticed this a few times now - Rider adds empty lines in the project files and I don't know why. I'll clean the `.csproj` files up.